portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -0,0 +1,47 @@
+package com.google.sps.servlets;
```

Comment:
Something to consider: It is also possible to use a single servlet (e.g. /comments) which supports both doGet() and doPost(). 

If the components change independently, then it usually cleaner to keep the logic separate. However if they change together, then grouping them together may be easier to maintain.

Reply:
Good idea. I combined the `NewCommentServlet` and the `ListCommentsServlet` into one `CommentServlet` - this makes more sense to me. I still have the `DeleteCommentServlet` separate because it has a different `doPost` function - let me know if there is a better way to do this. Thank you!

Reply:
Awesome! 

If you would prefer to include the delete method in the same servlet, there is an HTTP method called DELETE which is intended for requests that delete resources. Instead using a POST request in your javascript, you could change that to a DELETE request and then implement a doDelete method within this servlet.

for reference: https://docs.oracle.com/javaee/6/api/javax/servlet/http/HttpServlet.html#doDelete(javax.servlet.http.HttpServletRequest,%20javax.servlet.http.HttpServletResponse)